readme.md

```diff
@@ -21,12 +21,12 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://r-prykhodko.github.io/layout_stars/)
+- [TEST REPORT LINK](https://r-prykhodko.github.io/layout_stars/report/html_report/)
```

Comment:
You must add these links in the description of each PR so that we can check the UI

Now I have to copy the links manually and paste them into a separate browser window, this is inconvenient and takes time + you can conveniently see if the links work

And that's what I see when I open your link
<img width="1504" alt="Screenshot 2024-06-21 at 12 19 55" src="https://github.com/mate-academy/layout_stars/assets/94292654/eb2809f2-1dd4-4354-879f-2b4858b20ad2">

That what your Test link looks like
<img width="1458" alt="Screenshot 2024-06-21 at 12 21 08" src="https://github.com/mate-academy/layout_stars/assets/94292654/fb5be4ab-7003-43d5-8d43-4ea2ac683c32">


So I can't check either the UI or whether the tests passed
We will do this for every task, so it’s veeeery important that you learn how to deploy your work, if you can’t do it, please come to the Q&A, we will help you ;)